src/app/query/bitcoin/ordinals/use-inscriptions.query.ts

```diff
@@ -93,11 +93,13 @@ export function useTaprootInscriptionsInfiniteQuery() {
       let offset = pageParam?.offset || 0;
       // Loop through addresses until we reach the limit, or until we find an address with many inscriptions
       while (addressesWithoutOrdinalsNum < stopSearchAfterNumberAddressesWithoutOrdinals) {
-        const response = await fetchInscriptions(
-          Object.keys(addressesData),
-          offset,
-          inscriptionsLazyLoadLimit
-        );
+        const addresses = Object.keys(addressesData);
+
+        // add native segwit address to 1 chunk of addresses
+        if (fromIndex === 0) {
```

Comment:
don't much like this, but couldn't find good solution now, will try to find better place to insert native segwit address

Reply:
Not sure what this actually does? 🤔

Reply:
here I just add ns address to 1 chunk of taproot addresses